typescript/sdk/src/token/adapters/SealevelTokenAdapter.ts

```diff
@@ -297,11 +312,13 @@ export abstract class SealevelHypTokenAdapter
     });
 
     const setComputeLimitInstruction = ComputeBudgetProgram.setComputeUnitLimit(
-      {
-        units: TRANSFER_REMOTE_COMPUTE_LIMIT,
-      },
+      { units: TRANSFER_REMOTE_COMPUTE_LIMIT },
     );
 
+    const setPriorityFeeInstruction = ComputeBudgetProgram.setComputeUnitPrice({
```

Comment:
nit: would add link to the phantom priority fee docs https://docs.phantom.app/developer-powertools/solana-priority-fees#how-phantom-applies-priority-fees-to-dapp-transactions